itou/siae_evaluations/models.py

```diff
@@ -272,6 +309,10 @@ class Meta:
     def __str__(self):
         return f"{self.job_application}"
 
+    @property
+    def state(self):
+        return evaluation_enums.EvaluationJobApplicationsState.PENDING
```

Comment:
ça mérite un petit commentaire pour dire que actuellement c'est un work in progress, non ?